addons/mail/static/src/new/discuss/channel_member_list.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module **/
+
+import { Component } from "@odoo/owl";
+import { useMessaging } from "@mail/new/messaging_hook";
+import { PartnerImStatus } from "./partner_im_status";
+
+export class ChannelMemberList extends Component {
+    setup() {
+        this.messaging = useMessaging();
+        this.members = this.getMembersByStatus();
+        this.onlineMembers = this.members.onlines.sort((p1, p2) => (p1.name < p2.name ? -1 : 1));
+        this.offlineMembers = this.members.offlines.sort((p1, p2) => (p1.name < p2.name ? -1 : 1));
+    }
```

Comment:
It's best to use getters, because it's very likely to become unsync whenever reactive state changes (and the attributes are not updated accordingly)